dapr/clients/grpc/client.py

```diff
@@ -451,6 +455,59 @@ def save_state(
         return DaprResponse(
             headers=call.initial_metadata())
 
+    def execute_transaction(
+            self,
+            store_name: str,
+            operations: Sequence[TransactionalStateOperation],
+            metadata: Optional[MetadataTuple] = ()) -> DaprResponse:
```

Comment:
I would prefer to have this a mandatory field. 
@youngbupark thoughts?

Reply:
python-sdk exposes metadata only for grpc request. it is definitely optional so that user can pass the custom metadata to the grpc request. 

but we need to add another metadata for `transactional_metadata` for metadata in transactional request. it is also optional.

Reply:
OK. Adding transactional_metadata.